src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -445,6 +446,25 @@ public virtual int TupleElementIndex
         {
             get
             {
+                if (ContainingType.IsTupleType && ContainingType.IsDefinition)
```

Comment:
>ContainingType.IsDefinition [](start = 50, length = 27)

Why is it important that the containing type is the definition?

Reply:
In thinking about this question, I realized it's not. This implementation should handle the underlying fields in a constructed tuple. Will add test and fix

---
In reply to: [425344464](https://github.com/dotnet/roslyn/pull/44231#discussion_r425344464) [](ancestors = 425344464)